### DISCUSSION TITLE
Do you have plans to implement a calendar widget?

### DISCUSSION THREAD

**Discussion by nascin** [flet-dev/flet#199] [Ideas]
Do you have plans to implement a calendar widget, or is that something the developer should implement?

**Comment by FeodorFitsner** [comment]
Are you referring to any particular Flutter Calendar widget?

**Reply by nascin** [reply]
not really, but it would be interesting to include some.

**Reply by azhago** [reply]
Hi,
Very interested in a calendar widget too.
I found this one https://pub.dev/packages/syncfusion_flutter_calendar
